src/web3/networkConfig.ts

```diff
@@ -72,7 +72,7 @@ interface NetworkConfig {
   }
   celoTokenAddress: Address
   celoGasPriceMinimumAddress: Address
-  alchemyRpcUrl: Partial<Record<Network, string>>
+  alchemyRpcUrl: Record<Exclude<Network, Network.Celo>, string>
```

Comment:
tested this by deleting OP from alchemyRpcUrl and verifying that it now gives a compilation error where it didn't before